include/seqan3/alignment/configuration/detail.hpp

```diff
@@ -34,6 +32,7 @@ enum struct align_config_id : uint8_t
     on_result,    //!< ID for the \ref seqan3::align_cfg::on_result "on_result" option.
     parallel,     //!< ID for the \ref seqan3::align_cfg::parallel "parallel" option.
     result,       //!< ID for the \ref seqan3::align_cfg::result "result" option.
+    result_type,  //!< ID for the \ref seqan3::align_cfg::detail::result_type "result_type" option.
```

Comment:
This produces an error in the user documentation build, because it does not know about detail members. Try to work with `\if DEV`, but could be difficult in a table...

Reply:
all of these PR's have the problem, I added a working fix in https://github.com/seqan/seqan3/pull/2020

But seeing that multiple PRs need that, I'll extract that commit in a new PR.

Reply:
So I leave it as is ?

Reply:
yes